Dockerfile-dcrd

```diff
@@ -4,43 +4,63 @@
 
 FROM ubuntu:xenial
 
-LABEL description="Decred mainnet daemon"
-LABEL version="1.1"
-LABEL maintainer "john@netpurgatory.com"
+LABEL description="Decred mainnet daemon" \
+      version="1.2" \
+      maintainer="john@netpurgatory.com"
 
-ENV DCRD_RELEASE v1.0.5
-
-ENV TERM linux
-ENV USER decred
+ENV DCR_RELEASE="v1.0.5" \
```

Comment:
Combining these variables into one line also has no impact on final size.  Please put it back as it was.  I also don't like that when you do it this way you have to repeat things (like the username) rather than just reusing the variable the way I had it.